torchbenchmark/models/mistral_7b_instruct/metadata.yaml

```
@@ -0,0 +1,10 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 1
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+train_benchmark: false
+train_deterministic: false
+not_implemented:
+  - device: NVIDIA A10G
```

To disable CPU runs, you need to add the following line:
```suggestion
  - device: NVIDIA A10G
  - device: cpu
```